tests/unit/eblob_wrapper.cpp

```diff
@@ -0,0 +1 @@
+#include <eblob_wrapper.h>
```

Comment:
Why did you create `eblob_wrapper.cpp`?

Reply:
I wanted move method implementations here

Reply:
Ok, I'm not against at all. Assuming that the work in progress.